src/Symfony/Bundle/FrameworkBundle/Tests/DependencyInjection/Compiler/PropertyInfoPassTest.php

```diff
@@ -31,12 +32,16 @@ public function testServicesAreOrderedAccordingToPriority()
             new Reference('n3'),
         );
 
-        $container = $this->getMockBuilder('Symfony\Component\DependencyInjection\ContainerBuilder')->setMethods(array('findTaggedServiceIds'))->getMock();
+        $container = $this->getMockBuilder('Symfony\Component\DependencyInjection\ContainerBuilder')->setMethods(array('findTaggedServiceIds', 'getDefinition'))->getMock();
```

Comment:
We don't, these tests use reflection to check that the private `findTaggedServiceIds` is called, they could just register some services with the good tag in a concrete builder and check the services arguments/method calls after processing, which is the right thing to assert IMO as these passes just set arguments/calls on the collected services.